DOCUMENTATION.md

```diff
@@ -390,6 +390,22 @@ E.g : `{% image fancybox right clear group:travel image2.png http://google.fr/im
 |Height of thumbnail image (optional)| Height to the thumbnail image. If the thumbnail image is empty, height will be attached to thumbnail image created from original image. E.g : `300px` or `20%`.|
 |Title (optional)| Title of image displayed in a caption under image. `Alt` HTML attribute will use this title. E.g : `"A beautiful sunrise"`.|
  
+### Video
+The video tag is useful to add html5 mp4 videos. Sometimes used as more efficient "gifs" since in compairison, "gifs" are [way too huge](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/replace-animated-gifs-with-video/).
+Take a look at some examples here: [Video Tag Demo](https://benmitchinson.com/Video-Tag-Demo/)
```

Comment:
I will host a video, a thumbnail and update the demo site so we keep everything under the same repository. It will be easier to maintain and more reliable. I will give you these URLs once I took care of that.

Reply:
👍